### PR TITLE
fix linting and make some corrections

### PR DESCRIPTION
**Pull request by adamscarberry**
There's going to be some extra changed due to prettier differences (I use single quotes).  If we want to decide on common formatting that includes double quotes I can fix this before merging.

https://github.com/USACE/groundwork/issues/34

### CODE REVIEW

**Comment by jbkolze** [comment]
@adamscarberry , my main issue was actually with the .eslintrc.cjs file.  Mind if I push a commit containing some edits there to this or should I do it as a separate PR?

**Comment by adamscarberry** [comment]
@jbkolze Yea go ahead and include since it's linting related.  I would like to also include `eslint-plugin-jsx-a11y` in our devDependencies and configure in eslintrc.cjs, but didn't want to throw too much at once.

example: https://github.com/USACE/a2w-ui/blob/main/.eslintrc.cjs

**Comment by jbkolze** [comment]
@adamscarberry I threw in what I had in mind as a starting point.  Basically standard defaults with a couple tweaks to not drown in error hell in existing files (based on what I checked, at least).  Agree with jsx-a11y -- will let you handle that part since I don't have a lot of experience with it.

Feel free to adjust as needed / preferred.

**Comment by jbkolze** [comment]
Just for clarification -- I think the typescript linting stuff was a remnant from the initial groundwork build using styled-components and typescript.  Typescript should be ripped out completely at this point.  I think the .eslint was just the last piece that got missed.

**Comment by adamscarberry** [comment]
I think I'm done with changes on this one.

File: src/app-components/sub-site-banner.jsx
Comment (willbreitkreutz) [inline src/app-components/sub-site-banner.jsx:1]: lol, my regex went a little crazy there... thanks for catching that @adamscarberry

File: src/app-pages/documentation/app-shell/search.jsx
Comment (willbreitkreutz) [inline src/app-pages/documentation/app-shell/search.jsx:1]: doh

File: tailwind.config.js
Comment (willbreitkreutz) [inline tailwind.config.js:1]: What if i wanted two in there...

**Review by willbreitkreutz — APPROVED** [review]
Thanks for all the clean up work!

**Pull request merged**